src/dsp_tools/commands/create/models/parsed_ontology.py

```diff
@@ -25,7 +28,40 @@ class ParsedClass:
 @dataclass
 class ParsedProperty:
     name: str
-    info: dict[str, Any]
+    labels: dict[str, str]
```

Comment:
Not sure if we have discussed it already, but I like the concept of having type safety for the languages, like the example below. But of course, that's just a suggestion. If it turns out to be impractical, then let's stick with the generic `str`.

```python
@unique
class Languages(Enum):
    """Languages supported by DSP"""

    EN = "en"
    DE = "de"
    FR = "fr"
    IT = "it"
    RM = "rm"
```